src/ape/pytest/runners.py

```diff
@@ -133,7 +133,8 @@ def pytest_runtest_setup(self, item):
         """
         if (
             self.config_wrapper.isolation is False
-            or isinstance(item, pytest.DoctestItem)  # doctests don't have fixturenames
+            # doctests don't have fixturenames
```

Comment:
Wait. Are doctests actually running?

Reply:
Not us, but in Ape projects that have both

Reply:
Ah, would be sweet to have doc tests set up

Reply:
So would a ton of ice cream right now

Reply:
The issue is tracked here: https://github.com/ApeWorX/ape/issues/657
It would be a good amount of work I think getting those running how we want.